aria-practices.html

```diff
@@ -6279,6 +6279,39 @@ <h3>Indicating sort order with <code>aria-sort</code></h3>
+    <p>For example, given the HTML document</p>
+    <pre><code>&lt;!doctype html&gt;
+&lt;html&gt;
+ &lt;head&gt;
+  &lt;title&gt;Cookie policy&lt;/title&gt;
+ &lt;/head&gt;
+ &lt;body&gt;
+  &lt;p&gt;Cookies are delicious delicacies. &lt;button&gt;OK&lt;/button&gt;&lt;/p&gt;
+ &lt;/body&gt;
+&lt;/html&gt;</code></pre>
+    <p>The DOM tree includes element nodes closely resembling the markup, with a root <code>html</code> element that has two element children <code>head</code> and <code>body</code>, and so on.</p>
+    <p>The accessibility tree does not include the <code>head</code> or <code>title</code> elements, because the <code>head</code> element is hidden by default. The paragraph is included in the accessibility tree, with a text child node &quot;Cookies are delicious delicacies. &quot; and a child <code>button</code> element node, whose accessible name is &quot;OK&quot;.</p>
+    <p>CSS and ARIA can be used to control what is included in the accessibility tree, which is discussed in the following sections.</p>
+  </section>
+
+  <section id="excluding_subtrees_from_the_accessibility_tree">
+    <h2>Excluding subtrees from the accessibility tree</h2>
+    <p>An element that is hidden, either by default or by the author using CSS (in particular <code>display: none</code> and <code>visibility: hidden</code>), or using the HTML <code>hidden</code> attribute (which by default maps to CSS <code>display: none</code>), is not included in the accessibility tree. Its descendants are also not included. The element is hidden from all users, both visually and from assistive technologies.</p>
+    <p>For example, a <a href="#disclosure">disclosure</a> shows and hides content, and that content is intended to be hidden for all users, and could use CSS <code>display: none</code> or the HTML <code>hidden</code> attribute.</p>
+    <p>Using <code>aria-hidden</code> with the value <code>true</code> excludes the element and its descendants from the accessibility tree, just like hidden elements, but such elements are <em>not</em> hidden visually. The descendants are excluded from the accessibility tree even if an element in the excluded subtree sets <code>aria-hidden</code> to <code>false</code>.</p>
+    <p><code>aria-hidden</code> can be used to hide decorative or redundant content from screen readers, for example an inline SVG icon.</p>
+    <pre><code>&lt;a href=&quot;/&quot;&gt;
+ &lt;svg aria-hidden=&quot;true&quot; focusable="false"&gt;...&lt;/svg&gt;
+ Home
+&lt;/a&gt;</code></pre>
+    <p class="note">The <code>focusable="false"</code> attribute is included to work around a bug in some versions of Internet Explorer, where <code>svg</code> elements are focusable by default.</p>
+    <p class="warning">Only use <code>aria-hidden=&quot;true&quot;</code> when hiding content from only assistive technologies improves the user experience of assistive technologies.</p>
```

Comment:
```suggestion
    <p class="warning">Only use <code>aria-hidden=&quot;true&quot;</code> to hide content from assistive technologies if it improves the experience for assistive technology users.</p>
```

Sentence felt awkward, partly because AT don't have experiences - AT users do.